package.json

```diff
@@ -2,15 +2,15 @@
   "name": "reselect",
   "version": "5.1.0",
   "description": "Selectors for Redux.",
-  "main": "./dist/cjs/reselect.cjs",
```

Comment:
:question: What's the bundle size change from having separate prod/dev CJS artifacts here?

Reply:
Can't tell right now. I'll be home soon and I can give you some numbers if you want but from what I remember it was positive, that's why I did it.